Getting error cannot find module 'class-validator'

Hi,

I am getting a transpiling error from running "tsc" into my project:

```
node_modules/fireorm/lib/src/AbstractFirestoreRepository.d.ts:2:33 - error TS2307: Cannot find module 'class-validator'.

2 import { ValidationError } from 'class-validator';
```
I added "fireorm": "^0.13.1" as Dependency and "reflect-metadata": "^0.1.13" as PeerDependency.

Any tips on what this could be related?

--- a/src/AbstractFirestoreRepository.ts
+++ b/src/AbstractFirestoreRepository.ts
@@ -1,6 +1,6 @@
 import { plainToClass } from 'class-transformer';
 import { DocumentSnapshot, QuerySnapshot } from '@google-cloud/firestore';
-import { ValidationError } from 'class-validator';
+import { ValidationError } from './Errors/ValidationError';
 
 import {
   IEntity,

--- a/src/Batch/BaseFirestoreBatchRepository.ts
+++ b/src/Batch/BaseFirestoreBatchRepository.ts
@@ -5,7 +5,6 @@ import {
   CollectionMetadata,
   MetadataStorageConfig,
 } from '../MetadataStorage';
-import { ValidationError } from 'class-validator';
 import { FirestoreBatchUnit } from './FirestoreBatchUnit';
 
 export class BaseFirestoreBatchRepository<T extends IEntity> {

--- a/src/Batch/FirestoreBatchUnit.ts
+++ b/src/Batch/FirestoreBatchUnit.ts
@@ -2,7 +2,7 @@ import { IEntity, Instantiable } from '../types';
 import { Firestore, DocumentReference } from '@google-cloud/firestore';
 import { CollectionMetadata } from '../MetadataStorage';
 import { serializeEntity } from '../utils';
-import { ValidationError } from 'class-validator';
+import { ValidationError } from '../Errors/ValidationError';
 
 type BatchOperation<T extends IEntity> = {
   type: 'create' | 'update' | 'delete';

--- a/src/Errors/ValidationError.ts
+++ b/src/Errors/ValidationError.ts
@@ -1,0 +1,45 @@
+/**
+ * Copied from class-validator/validation/ValidationError.d.ts
+ */
+export declare class ValidationError {
+  /**
+   * Object that was validated.
+   *
+   * OPTIONAL - configurable via the ValidatorOptions.validationError.target option
+   */
+  target?: Object;
+  /**
+   * Object's property that haven't pass validation.
+   */
+  property: string;
+  /**
+   * Value that haven't pass a validation.
+   *
+   * OPTIONAL - configurable via the ValidatorOptions.validationError.value option
+   */
+  value?: any;
+  /**
+   * Constraints that failed validation with error messages.
+   */
+  constraints: {
+    [type: string]: string;
+  };
+  /**
+   * Contains all nested validation errors of the property.
+   */
+  children: ValidationError[];
+  contexts?: {
+    [type: string]: any;
+  };
+  /**
+   *
+   * @param shouldDecorate decorate the message with ANSI formatter escape codes for better readability
+   * @param hasParent true when the error is a child of an another one
+   * @param parentPath path as string to the parent of this property
+   */
+  toString(
+    shouldDecorate?: boolean,
+    hasParent?: boolean,
+    parentPath?: string
+  ): string;
+}

